subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt

```diff
@@ -131,8 +132,8 @@ object InstantExecutionReportPage : Component<InstantExecutionReportPage.Model,
             ),
             div(
                 attributes { className("left") },
-                h1("${model.totalProblems} instant execution problems were found"),
-                learnMore(),
+                h1("${model.totalProblems} configuration cache problems were found"),
+                learnMore(model.documentationLink),
```

Comment:
I have the same change in a branch along with some other report changes, maybe just change the locations for now instead of the locations and content.

Reply:
Actually, I think I like your implementation better, so I'll get rid of my change.